Converters/Rhino/Speckle.Converters.RhinoShared/ToSpeckle/Meshing/DisplayMeshExtractor.cs

```diff
@@ -76,4 +82,77 @@ public static RG.Mesh GetDisplayMesh(RhinoObject obj)
     joinedMesh.Append(renderMeshes);
     return joinedMesh;
   }
+
+  /// <summary>
+  /// Calculating optimal meshing parameter 'minimumEdgeLength' for the given geometry.
+  /// </summary>
+  private static double GetAccurateMinEdgeLegth(RG.GeometryBase geometry)
+  {
+    // adjust meshing parameters if Brep edges are too close to the document tolerance
+    double minEdgeLength = 0.05;
+    if (geometry is RG.Brep brep && brep.Edges.Any(x => x.GetLength() < minEdgeLength))
+    {
+      return 0;
+    }
+
+    return minEdgeLength;
+  }
+
+  /// <summary>
+  /// Extracting Rhino Mesh from Rhino GeometryBase after moving it to origin (if needed).
+  /// </summary>
+  public static RG.Mesh MoveToOriginAndGetDisplayMesh(
+    RG.GeometryBase geometry,
+    bool modelFarFromOrigin,
+    out RG.Vector3d? vectorToOriginalGeometry
+  )
+  {
+    vectorToOriginalGeometry = null;
+
+    // 1. General check: if Model is NOT far from origin (99% of Rhino models): extract meshes as usual
+    if (!modelFarFromOrigin)
```

Comment:
change these redundant checks, just check for specific geometry being far from origin